spmd/tensor/ops/pointwise_ops.py

```diff
@@ -259,6 +333,8 @@
     "aten.nextafter.default",
     "aten.nextafter.out",
     "aten.nextafter_.default",
+    "aten.ones_like.default",
```

Comment:
we already have ones_like in https://github.com/pytorch/tau/blob/eba6bdf1ec50cfa2d86d032b66da1a356b0eeefc/spmd/tensor/ops/tensor_ops.py#L94 which is not really a pointwise rule